src/os/bluestore/BlueStore.cc

```diff
@@ -6282,6 +6282,7 @@ void BlueStore::_txc_state_proc(TransContext *txc)
     case TransContext::STATE_PREPARE:
       txc->log_state_latency(logger, l_bluestore_state_prepare_lat);
       if (txc->ioc.has_pending_aios()) {
+	txc->is_pipelined_io = true;
 	txc->state = TransContext::STATE_AIO_WAIT;
 	_txc_aio_submit(txc);
 	return;
```

Comment:
I don't understand the last sentence.
Do you mean that i have to change the variable name 'is_pipelined_io' into 'safe_for_sync_commit or something'??